code/src/App.js

```diff
@@ -1,9 +1,92 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import { THOUGHTS_URL, LIKES_URL } from './reusable/urls'
+import MessageList from './components/MessageList'
+import MessageForm from './components/MessageForm'
 
 export const App = () => {
+const [thoughtList, setThoughtList] = useState([])
+const [thoughtNew, setThoughtNew] = useState('')
+
+  
+  const fetchThoughtList = () => {
+    fetch(THOUGHTS_URL)
+      .then(res => res.json())
+      .then(thoughts => {
+        setThoughtList(thoughts)
+      })
+      .catch((err) => console.log(err))
+    }
+
+  const onLikesIncrease = (id) => {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json'
+      }
+    }
+    fetch(LIKES_URL(id), options)
+      // .then(fetchThoughtList())
+      // .catch(err => console.log(err))
+    fetch(LIKES_URL(id), options)
+      .then(res => res.json())
+      .then(receivedThought => {
+        const updatedThoughtList = thoughtList.map(thought => {
+          if (thought._id === receivedThought._id) {
+            thought.hearts += 1
+          }
+          return thought
+        })
+        setThoughtList(updatedThoughtList)
+      })
+      .catch(err => console.log(err))
+  }
+
+  const onFormSubmit = (event) => {
+    if (thoughtNew.length < 4 || thoughtNew.length > 140) {
+      alert('Your messsage needs to be between 5 and 140 characters!')
+    } else {
+
```

Comment:
"I'm not doing any of the advanced goals" -adds character counter to text area- 

Lars, you did an extra here you sneaky guy! :P Very nice and clean too, great job adding an alert.